app/lib/methods/sendFileMessage.ts

```diff
@@ -1,27 +1,34 @@
 import { sanitizedRaw } from '@nozbe/watermelondb/RawRecord';
 import { settings as RocketChatSettings } from '@rocket.chat/sdk';
-import { FetchBlobResponse, StatefulPromise } from 'rn-fetch-blob';
 import isEmpty from 'lodash/isEmpty';
+import { FetchBlobResponse, StatefulPromise } from 'rn-fetch-blob';
+import { Alert } from 'react-native';
 
-import FileUpload from './helpers/fileUpload';
-import database from '../database';
-import log from './helpers/log';
 import { IUpload, IUser, TUploadModel } from '../../definitions';
+import i18n from '../../i18n';
+import database from '../database';
+import FileUpload from './helpers/fileUpload';
 import { IFileUpload } from './helpers/fileUpload/interfaces';
+import log from './helpers/log';
 
 const uploadQueue: { [index: string]: StatefulPromise<FetchBlobResponse> } = {};
 
-export function isUploadActive(path: string): boolean {
-	return !!uploadQueue[path];
+export function isUploadActive(path: string, rid: string): boolean {
+	const uploadPath = `${path}-${rid}`;
+	return !!uploadQueue[uploadPath];
 }
 
-export async function cancelUpload(item: TUploadModel): Promise<void> {
-	if (!isEmpty(uploadQueue[item.path])) {
+export async function cancelUpload(item: TUploadModel, rid: string): Promise<void> {
+	const uploadPath = `${item.path}-${rid}`;
+	if (!isEmpty(uploadQueue[uploadPath])) {
 		try {
-			await uploadQueue[item.path].cancel();
+			await uploadQueue[uploadPath].cancel();
 		} catch {
 			// Do nothing
 		}
+		delete uploadQueue[uploadPath];
+	}
+	if (item._raw) {
```

Comment:
You should _never_ use `_raw` directly.
It starts with an underscore for a reason.